Docs/ROADMAP.md

```diff
@@ -0,0 +1,328 @@
+entirely new functionality. (ie a crossbow that does less damage, but knocks the enemy back very far).
+
+#### Menus
+
+There are many menus outlined above in this document. We will need to build out and polish these menus where applicable, in order to deliver an intuitive and visually 
+appealing experience for the user.
+
+#### Music
+
+We will need a soundtrack that fits the game's mood and tone. We can use any music released CC-BY, or develop it outselves.
+This music will chiefly be broken into the following categories:
+- Opening menu music
+- Combat music
+- Victory music
+- Defeat music
+
+#### Sound effects
+
+We will need sound effects for just about every in-game action as well as audio cues when a player navigates through menus or
+earns points/experience/currency.
+
+#### Level design
+
+We will need to design levels for the arena encounters. All scenery and terrain will need to be modeled and textured.
+Each level should contain spawn points for players, enemies, and powerups. Interactive elements such as doors or destructable
+windows will also need to be developed in order to give
+
+#### User experience/balancing
+
+It will be critical to focus on delivering a good experience to our end users. To do this, we will need to allow external playtesting and respond
+to feedback provided. If a feature does not positively contribute to the user's experience, it should be modified or removed.
+
+One common point of contention is balancing the rewards for different play styles so that they all feel valid to the player. This is highly subjective and will require much consideration.
+
+#### Writing
+
+Any dialogue, copy, or decisions regarding lore/story will need to be made as cohesively and consistently as possible. The game should have a consistent set of themes that 
+it adheres to.
+
+#### 2D Art
+
+Many UI elements will require us to make images.
+
+#### Marketing/PR
+
+We will need to create promotional materials for the game and likely give periodic press releases or dev log entries for the progress of the game.
+
+### Timeline
+
+“A delayed game is eventually good, a bad game is bad forever.” 
+― Shigeru Miyamoto
+
+The intent of this timeline is to identify how many sprints the team can commit to for this project, and to make the development of the game fit that window of time.
+We can periodically stop to reconsider the timeline if we either decide we need to release sooner, or extend the timeline to release later. 
+[Scope creep](https://en.wikipedia.org/wiki/Scope_creep) is something we will need to avoid when possible.
+
+The schedule for epics is tentative, and will very likely change as we begin work and realize the need for more or less stories. The epics are listed in the order that shey should be started. Many of the sprints are blank, as they will be used to continue work on epics from previous sprints.
+
+#### Sprint 1 preparation August 15, 2019
+- Marketing/PR
```

Comment:
I think this would be really hard in sprint 1 without any game assets to help leverage this, and I think marketing would be more of a continuous thing over the course of the whole project.

Reply:
I wasn't entirely sure how to lay out this section. I wanted to communicate that these epics would start development at particular times. This would chiefly aim to stagger creation of stories and limit the scope of development to a couple aspects at a time. I was thinking that we could start social media communications at this point, but for the reason you pointed out, it might be easier to start the marketing once assets are in place.